apps/modernization-ui/src/pages/patient/profile/document/PatientProfileDocuments.spec.tsx

```diff
@@ -114,10 +113,6 @@ describe('when at least one document is available for a patient', () => {
         const dateCreated = await findByText(/10\/07\/2021/);
 
         expect(dateCreated).toHaveTextContent('10:01 AM');
```

Comment:
Thought this might fail on non EST systems. Changed my timezone and it still passes.

Reply:
ahhh [global-setup.test.js](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/global-setup.test.js)